### ISSUE TITLE
XmlException: For security reasons DTD is prohibited in this XML document

### ISSUE THREAD

**Issue by nicolas-raoul** [aegif/CmisSync#540]
This strange error happens with SharePoint 2013:

```
2015-03-17 16:32:35,668 [5] ERROR CmisSync.Lib.Sync.CmisRepo+SynchronizedFolder [(null)] - Could not crawl folder: /
DotCMIS.Exceptions.CmisConnectionException: Parsing exception! ---> System.Xml.XmlException: For security reasons DTD is prohibited in this XML document. To enable DTD processing set the DtdProcessing property on XmlReaderSettings to Parse and pass the settings into XmlReader.Create method.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.ThrowWithoutLineInfo(String res)
   at System.Xml.XmlTextReaderImpl.ParseDoctypeDecl()
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.XmlTextReaderImpl.Read()
   at System.Xml.XmlReader.MoveToContent()
   at System.Xml.XmlReader.IsStartElement()
   at DotCMIS.Binding.AtomPub.AtomPubParser.Parse()
   at DotCMIS.Binding.AtomPub.AbstractAtomPubService.Parse[T](Stream stream)
   --- End of inner exception stack trace ---
   at CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.ProcessRecoverableException(String logMessage, Exception exception)
   at CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlRemoteFolder(IFolder remoteSubFolder, String localFolder, IList`1 remoteFolders)
   at CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlRemote(IFolder remoteFolder, String localFolder, IList`1 remoteFiles, IList`1 remoteFolders)
   at CmisSync.Lib.Sync.CmisRepo.SynchronizedFolder.CrawlSync(IFolder remoteFolder, String localFolder)
```

**Comment by nicolas-raoul** [comment]
[This QA](http://stackoverflow.com/questions/13854068/dtd-prohibited-in-xml-document-exception) hints to "set the ProhibitDtd property on XmlReaderSettings to false and pass the settings into XmlReader.Create method."

**Comment by nicolas-raoul** [comment]
Customer confirms that this error is fixed by adding `settings.DtdProcessing = DtdProcessing.Ignore;` in `atompub-parser.cs`.

**Comment by nicolas-raoul** [comment]
Fixed in CmisSync 2.3.2
